src/components/Dropdown/Dropdown.tsx

```diff
@@ -114,6 +112,10 @@ function childrenHaveLabelComponent(children?: ReactNode): boolean {
 /**
  * `import {Dropdown} from "@chanzuckerberg/eds";`
  *
+ * Note: this component has been deprecated and will be removed in a future release.
+ * Please use the Select component instead for forms, and the Menu component for other
+ * types of popover-like dropdowns.
+ *
```

Comment:
Could you add `@deprecated` to the bottom of this component docstring and add the deprecated badge to `Dropdown.stories.tsx`?

Reply:
Thanks for adding the `@deprecated` tag! Can you also add the badge in `Dropdown.stories.tsx`?

Reply:
Yep, this is recorded [here](https://github.com/chanzuckerberg/edu-design-system/blob/next/src/components/Dropdown/Dropdown.stories.tsx#L16) which came in with my other popover ticket

Reply:
Awesome, thanks!